contracts/Pool.sol

```diff
@@ -1126,6 +1126,13 @@ contract Pool is Ownable, Pausable, ReentrancyGuard {
                         : _player.mostRecentSegmentPaid >= (_depositCountMemory - 1)
                 ));
     }
+    /**
+       Returns the maximum amount that can be redeemed from a strategy for a player/admin
+     */
+    function getRedemptionValue(uint256 _amountToRedeem, uint256 _totalAmount) internal returns(uint256) {
+        if (_amountToRedeem > _totalAmount) return _totalAmount;
+        return _amountToRedeem;
```

Comment:
can we please continue w/ the current styling where `if` statements are closed around brackers `{}` (even for one liners).